docs/pages/guides/extending-world.mdx

```diff
@@ -484,3 +488,567 @@ The easiest way to create the Solidity code is to use the MUD template:
+
+     /*
+      * Create a temporary wallet and a viem client for it
+      * (see https://viem.sh/docs/clients/wallet.html).
+      */
+     const burnerAccount = createBurnerAccount(networkConfig.privateKey as Hex);
+     const burnerWalletClient = createWalletClient({
+       ...clientOptions,
+       account: burnerAccount,
+     });
+
+     /*
+      * Create an observable for contract writes that we can
+      * pass into MUD dev tools for transaction observability.
+      */
+     const write$ = new Subject<ContractWrite>();
+
+     /*
+      * Create an object for communicating with the deployed World.
+      */
+     const worldContract = getContract({
+       address: networkConfig.worldAddress as Hex,
+       abi: IWorldAbi,
+       publicClient,
+       walletClient: burnerWalletClient,
+       onWrite: (write) => write$.next(write),
+     });
+
+     /*
+      * Sync on-chain state into RECS and keeps our client in sync.
+      * Uses the MUD indexer if available, otherwise falls back
+      * to the viem publicClient to make RPC calls to fetch MUD
+      * events from the chain.
+      */
+
+     const { components, latestBlock$, storedBlockLogs$, waitForTransaction } = await syncToRecs({
+       world,
+       config: mudConfig,
+       tables: resolveConfig(counterConfig).tables,
+       address: networkConfig.worldAddress as Hex,
+       publicClient,
+       startBlock: BigInt(networkConfig.initialBlockNumber),
+     });
+
+     /*
+      * If there is a faucet, request (test) ETH if you have
+      * less than 1 ETH. Repeat every 20 seconds to ensure you don't
+      * run out.
+      */
+     if (networkConfig.faucetServiceUrl) {
+       const address = burnerAccount.address;
+       console.info("[Dev Faucet]: Player address -> ", address);
+
+       const faucet = createFaucetService(networkConfig.faucetServiceUrl);
+
+       const requestDrip = async () => {
+         const balance = await publicClient.getBalance({ address });
+         console.info(`[Dev Faucet]: Player balance -> ${balance}`);
+         const lowBalance = balance < parseEther("1");
+         if (lowBalance) {
+           console.info("[Dev Faucet]: Balance is low, dripping funds to player");
+           // Double drip
+           await faucet.dripDev({ address });
+           await faucet.dripDev({ address });
+         }
+       };
+
+       requestDrip();
+       // Request a drip every 20 seconds
+       setInterval(requestDrip, 20000);
+     }
+
+     return {
+       world,
+       components,
+       playerEntity: encodeEntity({ address: "address" }, { address: burnerWalletClient.account.address }),
+       publicClient,
+       walletClient: burnerWalletClient,
+       latestBlock$,
+       storedBlockLogs$,
+       waitForTransaction,
+       worldContract,
+       write$: write$.asObservable().pipe(share()),
+     };
+   }
+   ```
+
+   </CollapseCode>
+
+1. Now that `Counter` is available again, undo the edits that removed it.
```

Comment:
Couldn't we add `Counter` to the config as the first step to avoid the steps that remove it from the client code to simplify the guide?

Reply:
Yes. Done.